website/docs/podman/gpu.md

```diff
@@ -210,6 +212,21 @@ $ nvidia-ctk cdi list
 
 More information as well as troubleshooting tips can be found [on the official NVIDIA CDI guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/cdi-support.html).
 
+4. If on a SELinux-enabled OS (e.g. Fedora family), configure SELinux policy:
+
+Check whether SELinux is installed and enabled:
+
+```sh
+$ getenforce
+```
+
+- If `getenforce` is not found, or its output is `Permissive` or `Disabled`, no action is need.
+- If the output is `Enforcing`, configure SELinux to allow containers device access:
+
+```sh
+$ sudo setsebool -P container_use_devices true
+```
```

Comment:
We can indent this command to align with the previous bullet point to which it applies.

Reply:
Yeah sure that'd be nice. But I'm not sure how to do this with the specific renderer we're using.

Reply:
Github's renderer for example won't let you do this sort of thing. Unless there's some arcane trick I'm not aware of.

- List item 0
- List item 1
- ```sh
some code here
```

Reply:
No worries, i will handle it once the PR gets merged.